pom.xml

```diff
@@ -196,34 +196,35 @@
     </repository>
   </repositories>
 
-  <profiles>
+   <profiles>
     <profile>
-      <id>release</id>
-      <properties>
-        <plugin.version.gpg>1.6</plugin.version.gpg>
-        <gpg.passphrase>${env.GPG_PASS}</gpg.passphrase>
-        <resume>false</resume>
-        <tag>${env.RELEASE_VERSION}</tag>
-        <releaseVersion>${env.RELEASE_VERSION}</releaseVersion>
-        <developmentVersion>${env.DEVELOPMENT_VERSION}</developmentVersion>
-        <arguments>-DskipTests</arguments>
-      </properties>
+      <id>community-action-maven-release</id>
       <build>
-        <pluginManagement>
-          <plugins>
-            <plugin>
-              <groupId>org.apache.maven.plugins</groupId>
-              <artifactId>maven-gpg-plugin</artifactId>
-              <configuration>
-                <gpgArguments>
-                  <arg>--pinentry-mode</arg>
-                  <arg>loopback</arg>
-                </gpgArguments>
-              </configuration>
-            </plugin>
-          </plugins>
-        </pluginManagement>
+        <plugins>
```

Comment:
:thinking: I wonder whether we need this at all. This profile is defined in https://github.com/camunda-community-hub/community-hub-release-parent/blob/0f52943897c35d8012966c9b087a507d79b33697/pom.xml#L102-L120

Here we overwrite some stuff, but nothing really important. 

https://github.com/camunda-community-hub/zeebe-process-generator/issues/95

Reply:
This is where I really wasn't sure.